### DISCUSSION TITLE
[10/10] Recommend me your favorite  macOS Software (or plug-in)

### DISCUSSION THREAD

**Discussion by hackroid** [NAIST-SE/AttractingContributors#839] [2023 Attraction Contributions]
Please share your recommendations for a set of daily-use utility software or plug-ins that enhance productivity on your Mac. These tools can significantly improve efficiency, so I'd love to hear about your favorite software and plug-ins!

Here is my repo: https://github.com/hackroid/AttractingContributors_EverydayMacSoftware

**Comment by KKKKKXY** [comment]
I added a MAC app into your list.(githubid: KKKKKXY)

**Comment by Sanaa3131** [comment]
I couldn't merge your pull request because of a branch conflict, do you know how to solve it ? BTW, I have already contributed to your file

**Comment by hackroid** [comment]
help needed

**Reply by WEI44ZHEYUAN** [reply]
急急急急急
